### DISCUSSION TITLE
Deleted Organization still visible on profile

### DISCUSSION THREAD

**Discussion by midischwarz12** [community/community#118432] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

There's a deleted organization that is still on my personal profile page. In settings > organization page, the organization is not visible. So it is impossible for me to leave or remove the organization. I have waited over a year for it to be automatically removed on its own, and it hasn't. Please help.

**Comment by xSomoy** [comment]
I am facing the same problem
![github](https://github.com/user-attachments/assets/a4a91447-6109-4cf8-820b-7097c5e45670)
